hieradata/rspec/role_jenkins.yaml

```
@@ -0,0 +1,186 @@
+---
+lookup_options:
+  "^profile::jenkinscontroller::jcasc$":
+    merge:
+      strategy: deep
+      merge_hash_arrays: true
+
+profile::jenkinscontroller::ci_fqdn: 'jenkins.role.respec.test.local'
```

```suggestion
profile::jenkinscontroller::ci_fqdn: 'jenkins.role.rspec.test.local'
```
SSame as above, typo?